src/Neo.Extensions/Neo.Extensions.csproj

```
@@ -15,7 +15,7 @@
   <ItemGroup>
     <InternalsVisibleTo Include="Neo" />
     <InternalsVisibleTo Include="Neo.IO" />
-    <InternalsVisibleTo Include="$(AssemblyName).Tests" />
+    <InternalsVisibleTo Include="Neo.UnitTests" />
```

is not Neo.Extensions.Tests?
```suggestion
    <InternalsVisibleTo Include="$(AssemblyName).Tests" />
```

---

I made `Neo.Extensions` all the methods `public`, no need to have `internal`.

---

@shargon